### ISSUE TITLE
unreadable says to transpile then transpiling fails too.

### ISSUE THREAD

**Issue by WW91Tm90U21hcnQ** [lelinhtinh/de4js#52]
Hello,
thanks for this project it did deobfuscate the javascript to a good extent. But if you could guide me with making it readable that would be perfect. What I did is, make the js deobfuscated and then put the output again and use unreadable option to try to make it readable. I got error when tried to put 
[https://privatebin.net/?1f84ee47a49d9494#CW6W36QsssEjegaLUKd1TLuz1ZnUyAwctcDc4QgSqpqZ](url)
pass: a2V5
~ link open only for 1 day

what I want to do is deobfuscate, modify code, then obfuscate back (preferably with obfuscator io).
Can you please help solve the error when trying to use unreadable option?

Error compiling input. Try to check 'Transpile to ES5' in settings:

Line 254: Parse error.
(
expected

on using transpile on jsnice it gives another error.

note: along with my needed modifications I'll modify the creator credits in own way (not removing but making it good enough) just to not look that bad.

**Comment by lelinhtinh** [comment]
Same #51 output syntax error. I'm still working on how to fix this.

For backup: [WW91Tm90U21hcnQ.txt](https://github.com/lelinhtinh/de4js/files/5542085/WW91Tm90U21hcnQ.txt)

In the output, find syntax error locations and predict the correct code. Eg:
```js
static async.authenticate() // static async authenticate()
get.free() // get free()
.substr 1; // .substr(1);
['init']() // init()
```
Result: [god-bless-you.txt](https://github.com/lelinhtinh/de4js/files/5542083/god-bless-you.txt)

**Comment by lelinhtinh** [comment]
Fixed at version 1.10.3 [result.txt](https://github.com/lelinhtinh/de4js/files/5542497/result.txt)
